libtransact/src/database/sqlite.rs

```diff
@@ -14,3 +14,778 @@
  * limitations under the License.
  * ------------------------------------------------------------------------------
  */
+
+use std::cell::RefCell;
+use std::collections::VecDeque;
+use std::rc::Rc;
+
+use r2d2_sqlite::SqliteConnectionManager;
+use rusqlite::{named_params, params};
+
+use super::{
+    Database, DatabaseCursor, DatabaseError, DatabaseReader, DatabaseReaderCursor, DatabaseWriter,
+};
+
+type SqliteConnection = r2d2::PooledConnection<SqliteConnectionManager>;
+
+#[derive(Clone)]
+pub struct SqliteDatabase {
+    pool: r2d2::Pool<SqliteConnectionManager>,
+}
+
+impl SqliteDatabase {
+    pub fn new(path: &str, indexes: &[&str]) -> Result<Self, DatabaseError> {
+        eprintln!("new database at {}", path);
```

Comment:
Is `eprintln!` used intentionally?

Reply:
Removed (though it was removed in a later commit).